hkmc2/shared/src/main/scala/hkmc2/MLsCompiler.scala

```diff
@@ -74,7 +74,7 @@ class MLsCompiler(preludeFile: os.Path):
       val parsed = mainParse.resultBlk
       val (blk, newCtx) = elab.importFrom(parsed)
       val low = ltl.givenIn:
-        codegen.Lowering(lowerHandlers = false)
+        codegen.Lowering(lowerHandlers = false, None) // TODO: properly hook up stack limit
```

Comment:
When passing in some generic constant, it's better to name it.
```suggestion
        codegen.Lowering(lowerHandlers = false, stackLimit = None) // TODO: properly hook up stack limit
```